### ISSUE TITLE
No accessory

### ISSUE THREAD

**Issue by rhuss2** [Cisien/homebridge-sense-power-meter#1]
In HOOBS I have installed the Sense plugin and configured it with my username and password. According to the logs, it is loading and getting data. But I do not see anything in the Apple Home app when I open it, and I have tried adding an accessory but nothing shows up.
Is there something I am missing with this plug-in?

**Comment by wtwolfe** [comment]
It shows up like a light and that was muy experience as well...I uninstalled it.

[William W. - Chat @ Spike](https://spikenow.com/r/a/?ref=spike-organic-signature&_ts=rezvu)	[rezvu]

On November 6, 2020 at 16:34 GMT, rhuss2 <notifications@github.com> wrote:

In HOOBS I have installed the Sense plugin and configured it with my username and password. According to the logs, it is loading and getting data. But I do not see anything in the Apple Home app when I open it, and I have tried adding an accessory but nothing shows up.
Is there something I am missing with this plug-in?
